docs/src/use/configure/configuration-files.md

```diff
@@ -1,451 +1,396 @@
 ---
-title: Configuration Files
+title: Configuration Files (New)
 eleventyNavigation:
     key: configuration files
     parent: configure
-    title: Configuration Files
-    order: 2
+    title: Configuration Files (New)
+    order: 1
```

Comment:
Is `(New)` intentional? Since this now becomes the default config format, perhaps there's no longer a need to mark it as "new".